### ISSUE TITLE
Multiple Bugs

### ISSUE THREAD

**Issue by Kaizodude** [KamenkoTV/Qlauncher-mcpc-android#1]
Don't know if you are still updating / working on this, Works great with Pixel 2XL. But there are some bugs
-If you have a rounded screen it doesn't use the entire thing
-Buggy when you try to move / hit at the same time
-All blocks and entities turn black when a mob takes damage.

**Comment by artdeell** [comment]
This thing is made using source code of Boardwalk, and Boardwalk has same issues. If KamenkoTV will update souces of program from boardwalk 2.0 repository, some of this bugs will be fixed.

**Comment by artdeell** [comment]
So, you can try to build boardwalk 2.0 or wait.

**Comment by khanhduytran0** [comment]
This is a 'variant' seems to be different code from Boardwalk, and this is an version made by me since 1 year ago, but @KamenkoTV take the source code without credits. Although the decompiled code was ugly and you can't compile it.

**Comment by Bandito722** [comment]
> This is a 'variant' seems to be different code from Boardwalk, and this is an version made by me since 1 year ago, but @KamenkoTV take the source code without credits. Although the decompiled code was ugly and you can't compile it.

Do you have the original code?
My friends and i we really wanna make all possible to "revive" all of this.
(I don't speak English, so if you do not understand nothing tha i'm saying, i`m sorry).

**Comment by khanhduytran0** [comment]
No, even with backup code and the original APK. The oldest backup is 1.0.4 (I don't remember if I removed older backups). I didn't save the source on Github.

**Comment by Bandito722** [comment]
> No, even with backup code and the original APK. The oldest backup is 1.0.4 (I don't remember if I removed older backups). I didn't save the source on Github.
> EDIT: It's not "dead", it's still update on YouTube under a different name.

A different name? Can you tell me the name, please?

**Comment by khanhduytran0** [comment]
See https://youtu.be/PgtNWWO8L04
